index.html

```diff
@@ -239,13 +239,15 @@ <h1>Skills</h1>
                 </form>
             </section>
         </section>
+        <footer>
```

Comment:
- [ ] nice work. for it to look more like the Figma design, Please add the extra space under the email in the contact section 
- [ ] kindly reduce the text area, and please respect the Figma margins for the `fullname, email and message`
- [ ] kindly reduce the email font size so it matches the Figma design
yours
![Screenshot (641)_LI](https://user-images.githubusercontent.com/65298048/206547812-b59cc999-b6ce-49d4-85e7-dfb2ec81ce85.jpg)
figma
![Screenshot (642)](https://user-images.githubusercontent.com/65298048/206547858-71015413-22fc-41a8-8a98-893ee27cc9ce.png)